### PR TITLE
Update c-quiz.md

### PR DESCRIPTION
**Pull request by tripathisharad**
Hi I have added a c programming question please add it with hacktoberfest accepted tag thankyou.

### CODE REVIEW

File: c-(programming-language)/c-quiz.md [hunk 1]
```diff
@@ -1070,3 +1070,28 @@ int main() {
 - [ ] 4
 - [ ] 2
 - [ ] 1
+
+
+ #### Q70. What will be the output of the following code snippet?
```
Comment (Ebazhanov) [inline c-(programming-language)/c-quiz.md:1075]:
```suggestion
 #### Q70. What will be the output of the following code snippet?
```

File: c-(programming-language)/c-quiz.md [hunk 2]
```diff
@@ -1070,3 +1070,28 @@ int main() {
...
+    printf("Value of b is: %d",b);  
+    return 0;  
+}  
+void change(int x,int y)  
+{  
+    x=13;  
+    y=17;  
+}  
+
+output --
```
Comment (Ebazhanov) [inline c-(programming-language)/c-quiz.md:1093]:
```suggestion
 #include <stdio.h>  
void change(int,int);  
int main()  
{  
    int a=10,b=20;  
    change(a,b); //calling a function by passing the values of variables.  
    printf("Value of a is: %d",a);  
    printf("\n");  
    printf("Value of b is: %d",b);  
    return 0;  
}  
void change(int x,int y)  
{  
    x=13;  
    y=17;  
}  

output --
```
you have to put this code inside backtick to make it properly visible in *.md file

File: c-(programming-language)/c-quiz.md [hunk 3]
```diff
@@ -1070,3 +1070,28 @@ int main() {
...
+[1.]10,20
+[2.]10,10
+[3.]20,20
+[4.]20,10
```
Comment (Ebazhanov) [inline c-(programming-language)/c-quiz.md:1097]: this is not typical format for quiz answers, please take a look at examples, thanks

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
@tripathisharad left comments

**Review by tripathisharad — COMMENTED** [review]
Hi I have resolved all the issue now please merge it, and if there anything else then please let me know thankyou.

**Comment by tripathisharad** [comment]
Hi @Ebazhanov I have arranged the fields as you suggested thanks for your support. Please merge it with hacktoberfest thankyou.

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
@tripathisharad can you also please resolve merge conflicts, thanks

**Comment by Ebazhanov** [comment]
@tripathisharad ping

**Comment by tripathisharad** [comment]
I have checked everything is ok thank you

**Pull request closed without merging**